src/api/users/test/e2e/getUsersResponse.json

```diff
@@ -0,0 +1,262 @@
+[
```

Comment:
Why a static file vs. a function that generates this?  I'm OK with this, but it's an external dependency for the test that needs to be kept in sync with it.